### ISSUE TITLE
Packagist errors [composer]

### ISSUE THREAD

**Issue by gjedeer** [gjedeer/celery-php#61]
```
The massivescale/celery-php package of which you are a maintainer has
failed to update due to invalid data contained in your composer.json.
Please address this as soon as possible since the package stopped updating.

It is recommended that you use `composer validate` to check for errors when you
change your composer.json.

Below is the full update log which should highlight errors as
"Skipped branch ...":

[Composer\Repository\InvalidRepositoryException]: Some branches contained invalid data and were discarded, it is advised to review the log and fix any issues present in branches

Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">2.1.1</span>)
Importing tag 2.1.1 (2.1.1.0)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">2.1</span>)
<span style="color:black;background-color:yellow;">Skipped tag 2.1, "https://api.github.com/repos/gjedeer/celery-php/contents/composer.json?ref=390f94f52a89d622312796727a97b97f330426b1" does not contain valid JSON
Parse error on line 9:
...er/celery-php/"    "license": "BSD-2-Cl
---------------------^
Expected one of: 'EOF', '}', ':', ',', ']'</span>
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">2.0.2</span>)
<span style="color:black;background-color:yellow;">Skipped tag 2.0.2, tag (2.0.2.0) does not match version (2.0.0.0) in composer.json</span>
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">celery31</span>)
Importing branch celery31 (dev-celery31)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">drewpctest</span>)
<span style="color:white;background-color:red;">Skipped branch drewpctest, "https://api.github.com/repos/gjedeer/celery-php/contents/composer.json?ref=fce5d20f669ac8b88a70a26392c2ebddc39d4cc5" does not contain valid JSON
Parse error on line 9:
...er/celery-php/"    "license": "BSD-2-Cl
---------------------^
Expected one of: 'EOF', '}', ':', ',', ']'</span>

Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">master</span>)
Importing branch master (dev-master)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">peclonly</span>)
Importing branch peclonly (dev-peclonly)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">php_amqp-0.3</span>)
<span style="color:black;background-color:yellow;">Skipped branch php_amqp-0.3, no composer file</span>
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">php-amqplib</span>)
Importing branch php-amqplib (dev-php-amqplib)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">pull-39-test</span>)
Importing branch pull-39-test (dev-pull-39-test)
Reading composer.json of <span style="color:green;">massivescale/celery-php</span> (<span style="color:yellow;">redis</span>)
Importing branch redis (dev-redis)

```

**Comment by jdufresne** [comment]
This is caused by a lot of invalid JSON in composer.json across different branches.. I'll start cleaning this up.

**Comment by jdufresne** [comment]
I think you should consider deleting tag `2.1` as the composer.json is malformed and can't be used as a release. This has already been fixed in `2.1.1`. Compser sometimes scans these tags and branches, that is what produces these warnings.

**Comment by jdufresne** [comment]
This should now be fixed. If you're still experiencing the same errors, please let me know.
